### PR TITLE
Is typing

### PR DESCRIPTION
**Pull request by eb16**
#### :link: Trello board reference:

- [Mostrar si la otra persona está escribiendo](https://trello.com/c/bPm3gyv1/73-mostrar-si-la-otra-persona-est%C3%A1-escribiendo)

---

#### :pencil2: Description:

When the user in the dashboard types, the widget should receive `is typing..`. The other way around when the widget types.

---

#### :movie_camera: Screen record:

<p align="center">

</p>

---

#### :pushpin: Notes:

- The feature is not intended to be fully concurrent. 
- The debounce was set in `3000ms`.

---

### CODE REVIEW

File: src/constants/channels.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const Channels = {
```
Comment (AndresVidalBerriel) [inline src/constants/channels.js]: Nice!

File: src/context/chat/index.jsx [hunk 2]
```diff
@@ -35,11 +35,19 @@ const ChatContextProvider = ({ children }) => {
   }
 
   const handleIsTyping = response => {
-    const isWriting = response.is_writing === 'true'
+    const isWriting = response.is_writing === 'true' && response.user_id // TODO: Change this to boolean when backend changes it
     dispatch(setChatIsTyping(isWriting))
   }
 
-  const value = { state, dispatch, createChat, handleIsTyping }
+  const sendIsTyping = async isWriting => {
```
Comment (AndresVidalBerriel) [inline src/context/chat/index.jsx]: The case where chat is not assigned yet must be considered. Typing in this case results in a bad request to the API. It should not be more difficult than adding a condition to this function checking that ChatId is not undefined.

Comment (eb16) [reply src/context/chat/index.jsx]: Will do it! Thank you so much for the catch!! 💪

Comment (eb16) [reply src/context/chat/index.jsx]: Done! 🙌

File: src/constants/channels.js (hunk 1 above)
Comment (eb16) [reply src/constants/channels.js]: 💪

**Pull request merged**